src/commands/msetnx.json

```diff
@@ -6,23 +6,11 @@
         "since": "1.0.1",
         "arity": -3,
         "function": "msetnxCommand",
-        "command_flags": [
-            "WRITE",
-            "DENYOOM"
-        ],
-        "acl_categories": [
-            "STRING"
-        ],
-        "command_tips": [
-            "REQUEST_POLICY:MULTI_SHARD",
-            "RESPONSE_POLICY:AGG_MIN"
-        ],
```

Comment:
i suppose the CI failure was due to the empty array, so deleting it completely is a good idea.
but let's avoid re-formatting other fields (just causes blame log destruction for no reason)

Reply:
done